content/releases.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Releases
+layout: page
+
+---
+
+# Releases
+
+This page describes the release process as well as the currently planned release schedule for upcoming releases as well as the respective release schepherds. Release shepherds are chosen on a voluntary basis.
+
+## Release schedule
+
+Release cadence of first pre-releases being cut is 6 weeks.
+
+| release series | date of first pre-release (year-month-day) | release shepherd                            |
+|----------------|--------------------------------------------|---------------------------------------------|
+| v2.4           | 2018-09-06                                 | Goutham Veeramachaneni (GitHub: @gouthamve) |
+| v2.5           | 2018-10-18                                 | Frederic Branczyk (GitHub: @brancz)         |
+| v2.6           | 2018-11-29                                 | **searching for volunteer**                 |
+
+If you are interested in volunteering please create a pull request against the [prometheus/docs](https://github.com/prometheus/docs) repository and propose yourself for the release series of your choice.
+
+## Release shepherd responsibilities
+
+The release shepherd is responsible for the entire release series of a minor release, meaning all pre- and patch releases of a minor release. The process starts with the initial pre-release.
+
+* The first pre-release is scheduled according to the above schedule.
+* With the pre-release the release shepherd is responsible for running and monitoring a benchmark run of the pre-release for 3 days, after which, if successful, the pre-release is promoted.
+* Once a pre-release has been released, the `master` branch of the repository is frozen for any feature work, only critical bug fix work concerning the minor release is merged.
+* Pre-releases are done from `master`, after pre-releases are promoted to the stable release a `release-major.minor` branch is created.
+
+See the next section for details on cutting an individual release.
+
+## How to cut an individual release
+
+These instructions are currently valid for the Prometheus server, i.e. the [prometheus/prometheus repository](https://github.com/prometheus/prometheus). Applicability to other Prometheus repositories depends on the current state of each repository. We aspire to use release procedures as similar as possible.
```

Comment:
> We aspire to use release procedures as similar as possible.

Sounds off, maybe:
We aspire to unify the release procedures as much as possible?